api/datasets.go

```diff
@@ -73,4 +78,35 @@ func ConfigureDatasetsRouter(e *echo.Group, dldm *core.DeltaDM) {
 
 		return c.JSON(200, ads)
 	})
+
+	datasets.PUT("/:dataset_id", func(c echo.Context) error {
```

Comment:
Would you also make a quick update to the API docs, adding this endpoint? It's just a md file in `/docs/ap.md`, copy an existing section and update with this PUT api

Reply:
Docs should now be updated with new PUT endpoint.